src/clinical/clinical-service.ts

```diff
@@ -289,13 +292,65 @@ export const getClinicalSearchResults = async (
 };
 
 export const getClinicalErrors = async (programId: string, donorIds: number[]) => {
-  // 1. Get the migration errors for every donor requested...
+  // 1. Get the migration errors for every donor requested
   const migrationErrors = await getClinicalEntityMigrationErrors(programId, donorIds);
 
-  // 2. ...and now remove from the list all valid donors (fixed with submissions since the migration)
-  const validErrorRecords = await getValidRecordsPostSubmission(programId, migrationErrors);
+  // 2. Remove from the list all valid donors (fixed with submissions since the migration)
+  const validPostSubmissionErrors = await getValidRecordsPostSubmission(programId, migrationErrors);
+
+  // 3. Remove from the list all errors which match Program Exceptions
+  const programExceptions = await exceptionService.operations.getProgramException({ programId });
+  const entityExceptions = await exceptionService.operations.getEntityException({ programId });
+  const hasProgramExceptions = 'exception' in programExceptions;
+  const hasEntityExceptions = 'exception' in entityExceptions;
```

Comment:
`programExceptions` and `entityExceptions` are of types `Result<EntityException>` and `Result<ProgramException>` they'll have a `success` property which to me is more stable than checking for a key.
https://github.com/icgc-argo/argo-clinical/blob/develop/src/exception/error-handling.ts#L26

Reply:
Yes but Jon pointed out that they were implemented as `success: boolean` vs `success: true` which was causing me major headaches, which was why I turned to `in`

I've updated this in https://github.com/icgc-argo/argo-clinical/pull/1091/commits/3c752f965ff3e194f7802322ad2495530f50f29d

The Errors function now just uses `checkForProgramAndEntityExceptions`